src/EditorContainer.js

```diff
@@ -10,6 +10,8 @@ type EditorContainerProps = {
     program: Program,
     syntax: TextSyntax,
     mode: EditorMode,
+    runButtonDisabled: string,
```

Comment:
Let's make `runButtonDisabled` into a `boolean `and implement the logic to determine if it's disabled in App.js

    runButtonDisabled={this.state.dashConnectionStatus !== 'connected'}